Library/Homebrew/rubocops/shared/desc_helper.rb

```diff
@@ -64,6 +64,10 @@ def audit_desc(type, name, desc_call)
         name_regex = name.delete("-").split("").join('[\s\-]?')
         problem "Description shouldn't start with the #{type} name." if regex_match_group(desc, /^#{name_regex}\b/i)
 
+        if type == :cask && match = regex_match_group(desc, /\b(macOS|Mac( ?OS( ?X)?)?|OS ?X)\b/i)
+          problem "Description shouldn't contain the platform." unless match[1] == "MAC"
```

Comment:
```suggestion
          problem "Description shouldn't contain the platform." if match[1] != "MAC"
```

Is this always returned as uppercase?